MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+include Makefile AUTHORS CHANGES LICENSE NOTICE
+include *.rst
+include Dockerfile
+include gunicorn.conf.py
+include package.json
+graft bouncer/static
+graft bouncer/templates
+graft bouncer/scripts
+global-exclude __pycache__
+global-exclude *.py[co]
```

Comment:
Given that we are using Python 3 exclusively for Bouncer, do we still need the `*.py[co]` exclude?